src/administration/integrations/github.md

```diff
@@ -32,15 +31,16 @@ scopes:
 
 Copy the token and make a note of it (temporarily).
 
-Note that for the integration to work, your GitHub user needs to have
-permission to push code to the repository.
+> **Note:**
+> For the integration to work, your GitHub user needs to have permission to push code to the repository.
 
 ### 2. Enable the integration
 
-Note that only `project owner` or `project admin` can manage the integrations.
+> **Note:**
+> Only `project owner` or `project admin` can manage the integrations.
```

Comment:
I'd prefer to not over-use callout notes.  Is this change necessary?  The word "Note" doesn't imply that it needs to be a callout.